src/timer.rs

```diff
@@ -158,7 +156,7 @@ impl<T> Timer<T> {
         // Update the head slot
         self.wheel[slot] = token;
 
-        trace!("inserted timout; slot={}; token={:?}", slot, token);
+        println!("inserted timout; slot={}; token={:?}", slot, token);
```

Comment:
Crap, I forgot to change those back. Nice catch.

On Mon, Apr 25, 2016, 11:48 AM Herman J. Radtke III <
notifications@github.com> wrote:

> In src/timer.rs
> <https://github.com/carllerche/mio/pull/382#discussion_r60936978>:
>
> > @@ -158,7 +156,7 @@ impl<T> Timer<T> {
> >          // Update the head slot
> >          self.wheel[slot] = token;
> >
> > -        trace!("inserted timout; slot={}; token={:?}", slot, token);
> > +        println!("inserted timout; slot={}; token={:?}", slot, token);
>
> Did you intend to convert all the trace! macros to println!?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/carllerche/mio/pull/382/files/e8280e9e3c1bee8f72b269296a8ef13dc098e8a4#r60936978>
>